gencred/cloudbuild.yaml

```
@@ -0,0 +1,40 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: golang:$_GO_VERSION
+    args:
+    - go
+    - build
+    - -mod=readonly
+    - -a
+    - -installsuffix=cgo
+    - -o=./gencred/gencred
+    - ./gencred/cmd/gencred
+    env:
+    - CGO_ENABLED=0
+    - GOOS=linux
+    - GOARCH=amd64
+  - name: gcr.io/cloud-builders/docker
+    args:
+    - build
+    - --tag=gcr.io/$PROJECT_ID/gencred:$_GIT_TAG
+    - --tag=gcr.io/$PROJECT_ID/gencred:latest
+    - ./pkg/gencred
```

This should be the relative path to the directory with the `gencred` `Dockerfile`?
```suggestion
    - ./gencred
```

---

Copy paste oversight.
Fixed.